### PR TITLE
ENH: Update event sequencer to take more human readable event sequences

### PR DESCRIPTION
**Pull request by slactjohnson**
## Description
Updated the event sequencer to take more readable sequences. Now, instead of giving the event sequencer a list of four lists that contain the beam codes, beam deltas, fiducial deltas, and burst counts for each line, you give the sequencer a list of lists, where each sub-list represents a single line of the event sequence. For example:
> seq = [[182,12,0,0], [170,2,0,0], [169,0,0,0], ....]
> sequencer.sequence.put_seq(seq)

## Motivation and Context
This should make writing code to generate sequences for the event sequencer a little simpler and more straight forward. Also, I like it more.

## How Has This Been Tested?
Tested on MEC event sequencer. I basically wrote in the wrapper that I used in MEC for LU57 into the ophyd device. I wrote a wrapper because the old way was kind of annoying. I updated the tests for the sequencer as well, and they pass.

## Where Has This Been Documented?
I updated the docstrings to reflect these changes.

### CODE REVIEW

File: pcdsdevices/sequencer.py [hunk 1]
```diff
@@ -120,24 +123,11 @@ def show(self, num_lines=None):
...
-        else:
-            # Included for the edge case where someone instantiates the event
-            # sequence by itself, rather than as part of an event sequencer.
-            msg = "No sequence length was specified, and the event sequence\n"
-            msg += "is not a child of an event sequencer!\n"
-            msg += "Please specify a number of lines to show.\n"
-            raise ValueError(msg)
-
-        for line in seq_list[0:seq_length]:
+        for nline, line in enumerate(curr_seq):
```
Comment (klauer) [inline pcdsdevices/sequencer.py]: Perhaps:
```suggestion
        for line in curr_seq[:num_lines]:
```

File: tests/test_sequencer.py [hunk 2]
```diff
@@ -157,16 +157,15 @@ def plan():
...
     seq = SimSequencer('ECS:TST:100', name='seq')
 
-    dummy_sequence = [[i for i in range(0, 20)],
-                      [i for i in range(20, 40)],
-                      [i for i in range(40, 60)],
-                      [i for i in range(60, 80)]]
+    dummy_sequence = [[1,  2,  3,  4],
+                      [5,  6,  7,  8],
+                      [9, 10, 11, 12],
+                      [13, 14, 15, 16]]
```
Comment (klauer) [inline tests/test_sequencer.py]: You might consider adding tests for scenarios where the user mistakenly makes a list such as:
`[1, 2, 3, 4], [5, 6, 78], ...`

**Comment by ZLLentz** [comment]
Can I merge this?

**Comment by klauer** [comment]
@ZLLentz 👍

**Pull request merged**